CMakeLists.txt

```diff
@@ -87,6 +88,11 @@ else()
     set(NGEN_IS_MAIN_PROJECT OFF)
 endif()
 
+if(NGEN_WITH_COVERAGE)
+    add_compile_options(--coverage -O0)
```

Comment:
If this noticeably slows the testing process at all, it should probably be OK to use `-Og`

Reply:
I didn't notice it slowing down too much, but if you notice it on your end, then we can make this change 🙂